### ISSUE TITLE
OBS-NDI on Arch Linux Not Working

### ISSUE THREAD

**Issue by guitaristtom** [Palakis/obs-ndi#250]
Sooooooo weird thing with OBS and the OBS-NDI plugin... It works fine under Windows 8.1 on the exact same machine, but not under the Arch install on it or my personal laptop (it just has Arch installed).

I have the [obs-ndi-bin](https://aur.archlinux.org/packages/obs-ndi-bin/) AUR package along with OBS-Studio installed, but I have also tried compiling and installing the obs-ndi package from the Git Repo to no avail. 

I have also tried running OBS through a terminal window to check it's console output, from what I can tell, NDI starts up and is detected just fine. Another small oddity is that under Windows (on two different machines) I am able to see the NDI output on the same machine that is sending it, but I can't under Arch.

**Comment by scarygliders** [comment]
Perhaps I can help...

I set out yesterday to get this working such that my main PC running Arch was the video source, and my laptop also running Arch was the reciever/encoder/streamer.

For hours I couldn't get it to work. OBS/NDI on the laptop wasn't even listing any output on the PC. I realised at that point that I hadn't enabled Avahi on the laptop - it wasn't running. After I did that, OBS/NDI then listed the output from the PC. But I still wasn't receiving video. Hmmmm....

Another couple of hours frustration ensued. Eventually, I thought "do I need avahi up and running on the Arch source PC?"

Enabled and started avahi-daemon on the source PC and voila! Everything working as intended.

So if you haven't already, ensure that avahi-daemon is installed, enabled, and started on both systems.
